raiden/tests/integration/api/test_restapi.py

```diff
@@ -250,6 +250,7 @@ def test_api_query_our_address(api_server_test_instance):
     our_address = api_server_test_instance.rest_api.raiden_api.address
     assert get_json_response(response) == {"our_address": to_checksum_address(our_address)}
 
+
```

Comment:
nit: extra blank line I'm guessing was not intentional.

Reply:
lint asked for it.
Not important anyway as this was already merged into raiden itself.